### DISCUSSION TITLE
Buttons that execute Mission Control functions (e.g. Show Desktop or Mission Control etc)

### DISCUSSION THREAD

**Discussion by lemoran** [lawand-dot-io/taskbar#25] [Feature requests]
Add buttons for Mission Control and/or showing desktop, that may be useful to switch tasks and view desktop content.

**Comment by lawand-dot-io** [comment]
Hello. Thanks for the feature request. I split the request for a start menu into those two requests: https://github.com/lawand-dot-io/taskbar/discussions/31 and https://github.com/lawand-dot-io/taskbar/discussions/32

**Comment by lawand-dot-io** [comment]
As for Mission control and show desktop, it might be easy to implement. I'll keep it in mind for future versions :)

**Comment by lawand-dot-io** [comment]
Duplicate request closed: https://github.com/lawand-dot-io/taskbar/discussions/46

**Comment by lawand-dot-io** [comment]
Note that one temporary solution I personally use is to make the bottom left corner a hot corner which shows the desktop once the mouse cursor has reached it.

**Comment by lawand-dot-io** [comment]
Duplicate request closed: https://github.com/lawand-dot-io/taskbar/discussions/53

**Comment by lawand-dot-io** [comment]
Duplicate request closed: https://github.com/lawand-dot-io/taskbar/discussions/126

Consider making the show desktop button either trigger mission control show desktop, or optionally execute a minimize on all open windows
